astropy/units/equivalencies.py

```diff
@@ -800,7 +800,7 @@ def with_H0(H0=None):
 
     Parameters
     ----------
-    H0 : None or `~astropy.units.Quantity`
+    H0 : None or `~astropy.units.Quantity` ['Hz']
```

Comment:
If we're listing a unit, might as well be the regular one; I guess 'frequency' is OK too.

Reply:
Ah. thanks. I though I'd corrected all those. That's a mistake. It should be frequency. A future PR will enable units in xref.

Reply:
When that happens, km / s / Mpc would probably be the preferred annotation.